sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -554,6 +575,61 @@ def _pyro_mcmc(
 
         return samples.detach()
 
+    def _pymc_mcmc(
+        self,
+        num_samples: int,
+        potential_function: Callable,
+        initial_params: Tensor,
+        mcmc_method: str = "slice_pymc",
```

Comment:
what was your reasoning to use the slice sampler from pymc as default?

Reply:
no reason other than that originally the default for pyro samplers was also slice before. would probably make more sense for both pyro and pymc samplers to have NUTS as default